pkgs/applications/office/osmo/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, pkgconfig, gtk3, libxml2, gettext, libical, libnotify
-, libarchive, gtkspell3, webkitgtk, libgringotts, gsettings_desktop_schemas }:
+, libarchive, gtkspell3, webkitgtk, libgringotts, wrapGAppsHook }:
```

Comment:
the full `gsettings_desktop_schemas` is actually not required. `wrapGAppsHook` does all the magic needed for gtk3.